From 6147f6523fd3e82b1bed0ea5de1c23cff808a39f Mon Sep 17 00:00:00 2001
From: Moosa <moosa.baransi@babcomsoftware.com>
Date: Thu, 29 Dec 2011 09:26:16 +0200
Subject: [PATCH] added driver version

* This version includes new iPerf which reduces
  the CPU utilization.

Signed-off-by: Moosa <moosa.baransi@babcomsoftware.com>
---
 drivers/net/wireless/wl12xx/main.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/drivers/net/wireless/wl12xx/main.c b/drivers/net/wireless/wl12xx/main.c
index aae6a74..fbf754b 100644
--- a/drivers/net/wireless/wl12xx/main.c
+++ b/drivers/net/wireless/wl12xx/main.c
@@ -49,6 +49,7 @@
 #include "scan.h"
 
 #define WL1271_BOOT_RETRIES 3
+#define WLAN_DRIVER_VERSION "R4_SP2_03_00"
 
 static struct conf_drv_settings default_conf = {
 	.sg = {
@@ -2227,6 +2228,7 @@ power_off:
 	wl->state = WL1271_STATE_ON;
 	set_bit(WL1271_FLAG_IF_INITIALIZED, &wl->flags);
 	wl1271_info("firmware booted (%s)", wl->chip.fw_ver_str);
+	wl1271_info("Driver version: %s", WLAN_DRIVER_VERSION);
 
 	/* update hw/fw version info in wiphy struct */
 	wiphy->hw_version = wl->chip.id;
-- 
1.7.0.4


